diff --git a/config.def.h b/config.def.h
index 6f05dce..55e494c 100644
--- a/config.def.h
+++ b/config.def.h
@@ -17,7 +17,7 @@ static int borderpx = 2;
  * 5: value of shell in config.h
  */
 static char *shell = "/bin/sh";
-char *utmp = NULL;
+char *utmp = "/usr/bin/tmux";
 /* scroll program: to enable use a string like "scroll" */
 char *scroll = NULL;
 char *stty_args = "stty raw pass8 nl -echo -iexten -cstopb 38400";

